vendor/forms.py

```diff
@@ -68,8 +68,10 @@ def clean(self):
                 self.cleaned_data['name'] = product_names[0]
             else:
                 self.cleaned_data['name'] = _("Bundle: ") + ", ".join(product_names)
-
-        if self.data['terms'] == TermType.SUBSCRIPTION and not cleaned_data['term_details']:
+        
+        term_detail_defaults = [ False for default in list(offer_term_details_default().key()) if default not in cleaned_data['term_details']]
+        
+        if self.data['terms'] == TermType.SUBSCRIPTION and False in term_detail_defaults:
```

Comment:
due to my own lack of awareness with the business requirement, I'm not 100% clear why we need this validation, but I do have one question: shouldn't this be in the clean_term_details method?